src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/inference/SimpleNovelAdjacency.java

```diff
@@ -0,0 +1,155 @@
+package org.broadinstitute.hellbender.tools.spark.sv.discovery.inference;
+
+import com.esotericsoftware.kryo.DefaultSerializer;
+import com.esotericsoftware.kryo.Kryo;
+import com.esotericsoftware.kryo.io.Input;
+import com.esotericsoftware.kryo.io.Output;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+// TODO: 1/19/18 step 1 towards the comprehensive view for Sv type inference
+@DefaultSerializer(SimpleNovelAdjacency.Serializer.class)
+public final class SimpleNovelAdjacency {
+
+    private static final NovelAdjacencyAndInferredAltHaptype.Serializer narlSerializer = new NovelAdjacencyAndInferredAltHaptype.Serializer();
+    private static final ChimericAlignment.Serializer caSerializer = new ChimericAlignment.Serializer();
+
+    private final NovelAdjacencyAndInferredAltHaptype novelAdjacencyAndInferredAltHaptype;
+    private final List<ChimericAlignment> alignmentEvidence;
+
+
+    public NovelAdjacencyReferenceLocations getNovelAdjacencyReferenceLocations() {
+        return novelAdjacencyAndInferredAltHaptype.novelAdjacencyReferenceLocations;
+    }
+    public byte[] getAltHaplotypeSequence() {
+        return novelAdjacencyAndInferredAltHaptype.altHaplotypeSequence;
+    }
+    public List<ChimericAlignment> getAlignmentEvidence() {
+        return alignmentEvidence;
+    }
+
+
+    public SimpleNovelAdjacency(final NovelAdjacencyAndInferredAltHaptype novelAdjacencyReferenceLocations,
+                                final List<ChimericAlignment> alignmentEvidence) {
+        this.novelAdjacencyAndInferredAltHaptype = novelAdjacencyReferenceLocations;
+        this.alignmentEvidence = alignmentEvidence;
+    }
+
+    public SimpleNovelAdjacency(final Kryo kryo, final Input input) {
+        novelAdjacencyAndInferredAltHaptype = narlSerializer.read(kryo, input, NovelAdjacencyAndInferredAltHaptype.class);
```

Comment:
Check for null arguments or make it private or protected if Kryo can handle it.

Reply:
made `private`